content/core-features/authentication.textile

```diff
@@ -455,12 +455,11 @@ A revocation request must include one or more target specifiers which define the
 
 By default, supported target specifiers are:
 
-- clientId:<clientId> := For a string @clientId@, matches tokens that have a non-empty @clientId@ equal to that given.
-- revocationKey:<revocationKey> := Matches tokens that have a @revocationKey@ claim that matches the given @revocationKey@.
-
+- clientId := For a string @clientId@, matches tokens that have a non-empty @clientId@ equal to that given, for example: <code>targets: ["clientId:client1@example.com" ]</code>
+- revocationKey := Matches tokens that have a @revocationKey@ claim that matches the given @revocationKey@, for example: <code>targets: ["revocationKey:users.group1@example.com"]</code>
```

Comment:
Again there is some confusion around what is the key and what is the value in the description I think. We need to clarify this.